tldw_Server_API/app/services/acp_runtime_policy_service.py

```
@@ -176,6 +176,35 @@ async def build_resolution_metadata(
+            from tldw_Server_API.app.core.Agent_Client_Protocol.templates import (
+                resolve_for_session,
+            )
+            from tldw_Server_API.app.core.DB_Management.ACP_Sessions_DB import ACPSessionsDB
+
+            db = ACPSessionsDB()
+            result = resolve_for_session(db, session_id, persona_id, template_name)
+            if result is not None:
+                return result
+        except Exception:
+            # DB templates not available; fall through to flat lookup.
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)